plugins/contentstack/src/plugin.ts

```diff
@@ -117,14 +118,23 @@ registerDataPlugin(
                 )}&${buildHeaders()}`;
               };
 
-              const envName = `environment=${environmentName}`;
               if (options.entry) {
-                return buildUrl(
-                  `entries/${options.entry}?${envName}&${model.referenceSearchParams}`
+                const query = qs.stringify(
```

Comment:
I assume we're using this because [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) doesn't support IE, is that correct?

I don't know if we actually support IE, so you might be able to use that instead.

Reply:
I actually ended up replacing `qs` with `URLSearchParams` for convenience!